src/vdaf/prio3.rs

```diff
@@ -82,67 +83,67 @@ pub type Prio3Aes128CountVecMultithreaded = Prio3<
     16,
 >;
 
-impl<S, P, const L: usize> Prio3<CountVec<Field128, S>, P, L>
+/// Construct an instance of a CountVec VDAF with the given suite and the given number of
+/// aggregators. `len` defines the length of each measurement.
+pub fn new_prio3_aes128_count_vec<S, P, const L: usize>(
```

Comment:
* Since this is for the `Aes128` PRG (vzv. `aes128` in the function's name) I don't think it makes sense for `P` and `L` to be free.

* Why is `S`? generic? It seems like all of its generic parameters are already fixed by other things.

Reply:
"Suite" is a holdover from previous VDAF drafts, and it referred to a combination of a stream cipher and a key derivation function. There have been some changes since, but the PRG largely replaces its function.

Reply:
It looks like the generic constructor goes all the way back to #137, and I couldn't find any discussion about why the decision was made; I went ahead and switched to a non-generic CountVec constructor, with a separate multithreaded constructor.

I also removed "suite" from all constructor doccomments, thanks for the context!